.github/workflows/ci.yml

```diff
@@ -102,6 +105,7 @@ jobs:
           args: --package nixpacks --lib --test generate_plan_tests
       - name: List docker tests (prep test-docker)
         id: docker-prep
+        if: matrix.os == 'ubuntu-latest'
```

Comment:
Is there a reason we only run docker tests on ubuntu? Just curious

Reply:
yeah they break on windows bc no buildkit

Reply:
What about macos?